### PR TITLE
Added migration version if the project is using rails 5.x

### PR DESCRIPTION
**Pull request by an-nasir**
Hi, 
please review my request.

### CODE REVIEW

File: lib/friendly_id/migration.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
+require 'rails'
+
 migration_class =
-  if ActiveRecord::VERSION::MAJOR >= 5
-    ActiveRecord::Migration[4.2]
+  if Rails.version.start_with? "5"
```
Comment (parndt) [inline lib/friendly_id/migration.rb]: Won’t this break again with Rails 6?

Comment (an-nasir) [reply lib/friendly_id/migration.rb]: @parndt yea, nice pick. We can make it to check for starts with 5 OR > 5.

But Travis was failing that "Rails is not available" in next line Rails::Version.. 
Can you guide me on requiring rails at this point?

Comment (an-nasir) [reply lib/friendly_id/migration.rb]: That would be fine too, http://api.rubyonrails.org/classes/ActiveRecord/VERSION.html

**Comment by an-nasir** [comment]
Hi @parndt I've changed Rails to ActiveRecord but I'm getting this error in Travis CI
`Gem::LoadError: Specified 'postgresql' for database adapter, but the gem is not loaded. Add `gem 'pg'` to your Gemfile (and ensure its version is at the minimum required by ActiveRecord)`
Any guide, I think I have to change something in Gemfile.rails.x.x.rb but I'm not sure what?

**Comment by parndt** [comment]
@an-nasir can I get your thoughts on this alternative PR #859  please ?

**Comment by an-nasir** [comment]
This is to achieve different thing like it  won't add specific migration in
inheritance. And in rails 5.x it's required to inherit from a specific
migration rather than just Migration.
Lemme know your thoughts

**Comment by parndt** [comment]
Thanks 👍 merged to 9cd4ce2

**Pull request closed without merging**